phpstan.neon.dist

```diff
@@ -60,9 +50,9 @@ parameters:
             identifier: staticMethod.internal
             path: tests/*.php
 
-        - # in tests, we can instantiate internal classes
-            identifier: new.internalClass
-            path: tests/*.php
+        # - # in tests, we can instantiate internal classes
```

Comment:
@cyl3x oh no, I guess this can be removed instead commented?

Reply:
No, we only wanted to allow internals in tests, but PHPStan complained about unnecessary ignored errors. It's fine to leave it as it is I think :thinking:

Reply:
but is is commented now and has no effect? :thinking: https://github.com/shopware/SwagPayPal/blob/trunk/phpstan.neon.dist#L53-L55

Reply:
Indeed - PHPStan complained that this was an unnecessary ignored error. If I uncomment it, PHPStan tells me that it doesn't match something. I don't want to turn off `reportUnmatchedIgnoredErrors`, so I've just commented it out to show that it's something we may want to ignore